profile/profile.go

```diff
@@ -674,6 +674,29 @@ func numLabelsToString(numLabels map[string][]int64, numUnits map[string][]strin
 	return strings.Join(ls, " ")
 }
 
+// SetTag sets the specified key to the specified value for all samples in the
+// profile.
+func (p *Profile) SetTag(key string, value []string) {
```

Comment:
The field is called "Label", should SetTag and HasTag be SetLabel and HasLabel?

Reply:
Fixed.